openwisp_notifications/settings.py

```diff
@@ -37,6 +37,10 @@
     'openwisp-notifications/audio/notification_bell.mp3',
 )
 
+OPENWISP_NOTIFICATIONS_EMAIL_BATCH_INTERVAL = getattr(
```

Comment:
Let's shorten this, we usually do this in most modules, eg:

```suggestion
EMAIL_BATCH_INTERVAL = getattr(
```